core/block_dragger.ts

```diff
@@ -382,6 +382,7 @@ export class BlockDragger implements IBlockDragger {
 
     blockAnimation.disconnectUiStop();
     this.connectionPreviewer.hidePreview();
+    this.connectionPreviewer.dispose();
```

Comment:
If `.dispose` calls `.hidePreview`, can the latter call be omitted here?

Reply:
It could be, but that would mean we're requiring a side effect (`.dispose` must hide any previews). Calling both separately makes the API cleaner and easier to document.